pyproject.toml

```diff
@@ -0,0 +1,50 @@
+[tool.poetry]
```

Comment:
You should install dynamic-versioning for publish pypi.

https://github.com/m3dev/gokart/blob/master/pyproject.toml#L13-L16

Reply:
Added in ba418e5 .